newIDE/app/src/AssetStore/AssetCard.js

```diff
@@ -19,6 +19,11 @@ const styles = {
     verticalAlign: 'middle',
     pointerEvents: 'none',
   },
+  previewImagePixelated: {
+    width: '-webkit-fill-available',
```

Comment:
Is there a standard way for this? Like setting the width to 100 and setting a max height?